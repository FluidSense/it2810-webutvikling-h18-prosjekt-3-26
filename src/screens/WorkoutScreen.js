import React from 'react';
import { View, Text, Button, StatusBar } from 'react-native';

class WorkoutScreen extends React.Component {
    render() {
      return (
        <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center', backgroundColor: 'blue', }}>
          <Text>Workouts Screen</Text>
          <Button
            title="Go to Home Screen"
            onPress={() => {
              this.props.navigation.navigate('Home')
            }}
          />
          <Text> Fix yo Settings dawg</Text>
          <Button
            title="Go to Settings"
            onPress={() => {
              this.props.navigation.navigate('Settings')
            }}
          />
        </View>
      );
    }  
  }   

  export default WorkoutScreen;