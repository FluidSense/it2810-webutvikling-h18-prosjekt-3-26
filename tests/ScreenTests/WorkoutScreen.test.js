import React from 'react';
import WorkoutScreen from '../../src/screens/WorkoutScreen';

import renderer from 'react-test-renderer';


test('Renders correctly', () => {
  const tree = renderer.create(<WorkoutScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});