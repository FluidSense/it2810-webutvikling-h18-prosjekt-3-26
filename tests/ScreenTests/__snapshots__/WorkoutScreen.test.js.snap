// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "blue",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Workouts Screen
  </Text>
  <View
    accessibilityComponentType="button"
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Go to Home Screen
      </Text>
    </View>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
     Fix yo Settings dawg
  </Text>
  <View
    accessibilityComponentType="button"
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Go to Settings
      </Text>
    </View>
  </View>
</View>
`;
